Tommy-Spurlock/Remember_Me
# Dates show in order and Celeb page is now functional.

Birthdates show in order and celeb page shows three celebrity birthdays. 

to test:
-log in and add new birthday reminders.
      -they should be in order by todays date through next year.
-open side tab and click celebrities
      - should see a page with todays date and time at top of screen as well as three celebrities who were born today.
